src/modules/toolbox/components/drawToolContent/DrawToolContent.js

```diff
@@ -67,6 +70,10 @@ export class DrawToolContent extends AbstractToolContent {
 				return { ...model, tools: data };
 			case Update_FileSaveResult:
 				return { ...model, fileSaveResult: data.fileSaveResult };
+			case Update_CollapsedInfo:
+				return { ...model, collapsedInfo: data };
+			case Update_CollapsedStyle:
+				return { ...model, collapsedStyle: data };
 		}
 	}
 
```

Comment:
Also for this MvuElement we should add a test that verifies its initial model...